lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -50,7 +50,17 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
-  <button class="button button-primary button-block"
+  <div class="modal-footer-warning"
+    ng-show="modal.state.hasDrive() && modal.state.isSystemDrive(modal.state.getDrive())">
+      Etcher has identified this as a system drive
+  </div>
+  <button class="button button-block"
+    ng-class="{ 'label-danger': modal.state.hasDrive() && modal.state.isSystemDrive(modal.state.getDrive()),
+                'button-primary': modal.state.hasDrive() && !modal.state.isSystemDrive(modal.state.getDrive()) }"
```

Comment:
I'm not 100% sure, but I think we can remove the `modal.state.hasDrive()` check and only apply the button type based on whether the drive is system or not. If we indeed have no drive, `ngDisabled` will take precedence over the button style and do the right thing.